sql/driving_distance/_withPointsDD.sql

```diff
@@ -26,6 +34,33 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 -- pgr_withPointsDD
 -------------------
 
+--v3.6
+CREATE FUNCTION _pgr_v6withPointsDD(
```

Comment:
Please refer to my comment in https://github.com/pgRouting/GSoC-pgRouting/pull/302/files related to using `v4`/`v6` and creating new files/functions.

This is fine for now.